doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -649,40 +650,28 @@ responsibility is on the developer.
 Reviewer responsibilities
 +++++++++++++++++++++++++
 
+* Test the PR you are reviewing.  Follow the instructions next to the
+  "Merge pull request" button in the "command-line instructions" link
+  to checkout a local copy of the PR for testing.  They will look something
```

Comment:
"check out" (I believe "checkout" is just a noun in English even though it's a git command?).